src/components/Steps/Step/Step.module.scss

```diff
@@ -0,0 +1,145 @@
+@import "~@santiment-network/ui/mixins";
+
+.wrapper {
+  position: relative;
+  overflow: visible;
+}
+
+.container[role="button"] {
+  cursor: pointer;
+  transition: opacity 0.3s;
+}
+
+.tail {
+  position: absolute;
+  left: 12px;
+  top: 0;
+  height: 100%;
+  width: 1px;
+  padding: 28px 0 4px 0;
+
+  &::after {
+    height: 100%;
+    width: 1px;
+    content: "";
+    display: inline-block;
+    border-radius: 1px;
+    transition: background 0.3s;
+    background-color: var(--porcelain);
+  }
+}
+
+.contentWrapper {
+  display: flex;
+}
+
+.content {
+  max-width: 340px;
+  overflow: hidden;
+}
+
+.title {
+  position: relative;
+  color: var(--title-color, var(--rhino));
+
+  @include text("body-2");
+}
+
+.description {
+  padding-bottom: 16px;
+  color: var(--description-color, var(--waterloo));
+
+  @include text("body-3");
+}
+
+.iconWrapper {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 24px;
+  height: 24px;
+  border-radius: 100%;
+  font-size: 14px;
+  margin-right: 10px;
+  background-color: var(--icon-background-color, var(--waterloo));
+  color: var(--icon-color, var(--white));
+}
+
+.icon {
+  margin-left: 1px;
+  color: currentColor;
+}
+
+.disabled {
+  &[role="button"] {
+    cursor: initial;
+  }
+
+  .title {
+    --title-color: var(--waterloo);
+  }
+
+  .description {
+    --description-color: var(--casper);
+  }
+
+  .iconWrapper {
+    --icon-background-color: var(--mystic);
+    --icon-color: var(--waterloo);
+  }
```

Comment:
`[D-2-1]` Described issue was not resolved. There is no need for nested selectors, since `css variables` defined in a parent rule are also available for child elements.